Plugins/Vorlon/plugins/domExplorer/vorlon.domExplorer.ts

```diff
@@ -370,19 +373,12 @@ module VORLON {
         private _generateColorfullLink(link: HTMLAnchorElement, receivedObject: any): void {
             this._appendSpan(link, "nodeName", receivedObject.name);
             
-            if (receivedObject.id) {
-                this._appendSpan(link, "nodeAttribute", " id");
-                this._appendSpan(link, "nodeTag", "=\"");
-                this._appendSpan(link, "nodeValue", receivedObject.id);
-                this._appendSpan(link, "nodeTag", "\"");
-            }
-
-            if (receivedObject.classes) {
-                this._appendSpan(link, "nodeAttribute", " class");
-                this._appendSpan(link, "nodeTag", "=\"");
-                this._appendSpan(link, "nodeValue", receivedObject.classes);
-                this._appendSpan(link, "nodeTag", "\"");
-            }
+            receivedObject.attributes.forEach(function(attr){
+                var node = document.createElement('span');
+                node.className = 'nodeAttribute';
+                node.innerHTML = '<span>' + attr[0] + '</span><span>' + attr[1] + '</span>'; 
```

Comment:
Correct, this prints the entire content of all attributes - is that what's
desired?
On May 12, 2015 2:00 PM, "David Catuhe" <notifications@github.com> wrote:

> In Plugins/Vorlon/plugins/domExplorer/vorlon.domExplorer.ts
> <https://github.com/MicrosoftDX/Vorlonjs/pull/56#discussion_r30179789>:
>
> > -                this._appendSpan(link, "nodeAttribute", " id");
> > -                this._appendSpan(link, "nodeTag", "=\"");
> > -                this._appendSpan(link, "nodeValue", receivedObject.id);
> > -                this._appendSpan(link, "nodeTag", "\"");
> > -            }
> > -
> > -            if (receivedObject.classes) {
> > -                this._appendSpan(link, "nodeAttribute", " class");
> > -                this._appendSpan(link, "nodeTag", "=\"");
> > -                this._appendSpan(link, "nodeValue", receivedObject.classes);
> > -                this._appendSpan(link, "nodeTag", "\"");
> > -            }
> > +            receivedObject.attributes.forEach(function(attr){
> > +                var node = document.createElement('span');
> > +                node.className = 'nodeAttribute';
> > +                node.innerHTML = '<span>' + attr[0] + '</span><span>' + attr[1] + '</span>';
>
> you are not generating ="...", correct?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/MicrosoftDX/Vorlonjs/pull/56/files#r30179789>.
>